tests/languages/tt2/string-interpolation_feature.test

```diff
@@ -0,0 +1,21 @@
+[% "Hello, $name!" %]
+
+----------------------------------------------------
+
+[
+	["tt2",
+		[
+			["delimiter", "[%"],
+		 	["double-quoted-string",
+			 	["\"Hello, ",
+				 ["variable", "$name"],
+				 "!\""]

```

Comment:
Indentation seems a bit off here too.
Something like the following would be easier to read:
```
["double-quoted-string", [
	"\"Hello, ",
	["variable", "$name"],
	"!\""
],
```